src/controllers/hid/hidcontroller.h

```diff
@@ -8,7 +8,7 @@
 #ifndef HIDCONTROLLER_H
 #define HIDCONTROLLER_H
 
-#include <hidapi.h>
+#include <hidapi/hidapi.h>
```

Comment:
Does this work in both cases? (When using the system lib (make sure you rename the included one to see if it still builds,) as well as when using the included one (make sure the system -dev lib is not installed and see if it still builds.))